routers/api/v1/repo/issue.go

```diff
@@ -218,7 +218,7 @@ func CreateIssue(ctx *context.APIContext, form api.CreateIssueOption) {
 		form.Labels = make([]int64, 0)
 	}
 
-	if err := issue_service.NewIssue(ctx.Repo.Repository, issue, form.Labels, assigneeIDs, nil); err != nil {
+	if err := issue_service.NewIssue(ctx.Repo.Repository, issue, form.Labels, nil); err != nil {
```

Comment:
Just a thought. I imagine that the assignee creation and webhook triggering calls could be moved inside `issue_service.NewIssue()`, unless there are other scenarios I fail to see.